src/pages/de/install/manual.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Manuelle Installation von Astro
+description: So installierst du Astro manuell mit NPM, PNPM oder Yarn.
+layout: ~/layouts/MainLayout.astro
+setup: import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
+---
+Bist du bereit, Astro zu installieren? In dieser Installationsanleitung findest du alle Informationen, um direkt loszulegen.
+
+#### Voraussetzungen
+
+- **Node.js** - `14.15.0`, `v16.0.0` oder höher.
+- **Texteditor** - Wir empfehlen [VS Code](https://code.visualstudio.com/) mit unserer [offiziellen Astro-Erweiterung](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode).
+- **Terminal** - Astro wird über seine Befehlszeilenschnittstelle (CLI) gesteuert.
+
+<InstallGuideTabGroup />
+
+#### So funktioniert die manuelle Installation
+
+Falls du keine [automatische Installation](/de/install/auto/) mit unserem Assistenten `create-astro` durchführen möchtest, kannst du dein Astro-Projekt mit dieser Anleitung selbst einrichten.
+
+
+## 1. Erstelle ein Projektverzeichnis
+
+Erstelle ein leeres Verzeichnis mit dem Namen deines Projekts und wechsle dann hinein.
+
+```bash
+mkdir mein-astro-projekt
+cd mein-astro-projekt
+```
+
+Sobald du dich in deinem neuen Verzeichnis befindest, erstelle die Projektdatei `package.json`. Damit verwaltest du die für dein Projekt erforderlichen Pakete, einschließlich Astro. Wenn du mit diesem Dateiformat nicht vertraut bist, führe den folgenden Befehl aus, um die Datei für dich erstellen zu lassen:
+
+```bash
+npm init --yes
+```
+
+
+## 2. Installiere Astro
+
+Füge zuerst Astro zu den erforderlichen Paketen deines Projekts hinzu:
+
+```bash
+npm install astro
+```
+
+Ersetze dann jegliche Platzhalter im Abschnitt `scripts` der Datei `package.json` mit folgenden Astro-Skripts:
+
+```diff
+  "scripts": \{
+-    "test": "echo \"Error: no test specified\" && exit 1"
++    "dev": "astro dev",
++    "start": "astro dev",
++    "build": "astro build",
++    "preview": "astro preview"
+  },
+```
+
+Du kannst diese Skripts später verwenden, um Astro zu starten und seine verschiedenen Befehle auszuführen.
+
+
+## 3. Erstelle deine erste Seite
+
+Verwende deinen Texteditor, um im Projektverzeichnis eine neue Datei mit dem Pfad `src/pages/index.astro` zu erstellen. Sie wird die erste Astro-Seite deines Projekts.
+
+Kopiere im Rahmen dieser Anleitung den folgenden Code-Schnipsel (einschließlich der Bindestriche `---`) und füge ihn in deine neue Datei ein:
+
+```astro
+---
+// Willkommen bei Astro! Alles innerhalb dieses mit drei Bindestrichen
+// vom Rest getrennten Abschnitts nennt man "Komponenten-Frontmatter".
+// Der Code hier wird nie im Browser ausgeführt.
+console.log('This runs in your terminal, not the browser!');
```

Comment:
I would also translate the console message.
Example: "Das wird in deinem Terminal ausgeführt, nicht im Browser!"